packages/docs/src/content/charts/01.bar/05.echarts.mdx

```diff
@@ -0,0 +1,230 @@
+---
+nav_group: "Bar"
+nav_order: 5
+title: "Echarts"
+description: "A headless-chart example inspired by Echarts"
+files: 
+  App.js: |
+    import Widget from "@meursyphus/flitter-react";
+    import { BarChart } from "@meursyphus/headless-chart";
+    import {
+      Text,
+      Border,
+      BoxDecoration,
+      Column,
+      Container,
+      EdgeInsets,
+      Row,
+      SizedBox,
+      TextStyle,
+      Padding,
+      MainAxisSize,
+      MainAxisAlignment,
+      Stack,
+      Positioned,
+      FractionalTranslation,
+      Offset,
+      CustomPaint,
+    } from "@meursyphus/flitter";
+
+
+    const data = {
+       labels: [
+         "Mon",
+         "Tue",
+         "Wed",
+         "Thu",
+         "Fri",
+         "Sat",
+         "Sun"
+       ],
+       datasets: [
+         {
+            legend: "",
+            values: [120, 200, 150, 80, 70, 110, 130]
```

Comment:
look great. but could you redesign more like this?
<img width="1137" alt="image" src="https://github.com/user-attachments/assets/ecd61dd8-c0d3-4d63-adff-22af74fc5c38" />

there is no yGridLine and bar color is not 'blue'

Reply:
and you should focus xTicks position on this screenshot

there is slight difference between your chart and echart

Reply:
I removed yGridLine by applying transparent. And I also fixed chart color and chart xTicks to make chart more similart to echarts.